tutorials/intro_to_api/projects.md

```diff
@@ -126,8 +126,8 @@ project.close() # <-- close the project
 task2 = simulator.run(input_dir=input_dir,
                       sim_config_filename="params.txt")
 
-print(task1.get_info()['project']) # "my_xbeach_project"
-print(task2.get_info()['project']) # "userab1cdef2" (default project)
+print(task1.get_info().to_dict()['project']) # "my_xbeach_project"
+print(task2.get_info().to_dict()['project']) # "userab1cdef2" (default project)
```

Comment:
Not for now (because we are fixing a tutorial) but I
I think needs to be made easier for the user. We can encapsulate a lot of this syntax:
What about this:

```
task2.get_info('project')
```

or just

```
task2.get_project()
```

Reply:
LGTM! I also made some changes to this part of the code in another PR, but used 'task2.get_info().project' instead.